src/sql/parts/query/editor/queryResultsView.ts

```diff
@@ -223,6 +226,23 @@ export class QueryResultsView extends Disposable {
 				this._panelView.pushTab(this.topOperationsTab);
 			}
 		}
+
+		// restore query model view tabs
+		this.input.state.visibleTabs.forEach(tabId => {
+			if (tabId.startsWith('querymodelview;')) {
+				// tab id format is 'tab type;title;model view id'
+				let parts = tabId.split(';');
+				if (parts.length === 3) {
```

Comment:
log an error if it is not 3, otherwise it will be hard to figure out the reason.

Reply:
we build this string.  it is not user provided.  I don't see a great way to surface the error here and since this isn't user provided, it doesn't seem necessary.